.gitignore

```diff
@@ -1,5 +0,0 @@
-/vendor
```

Comment:
Now you totally removed the .gitignore in the repository.

Reply:
@Korko thank you, how do you remove the single file from the pull request?

Reply:
Not sure how using the github interface but in git, should be simple:
`git fetch
git checkout origin/master -- .gitignore`
should reset the file to the master version

Reply:
> Not sure how using the github interface but in git, should be simple:
> `git fetch`
> `git checkout origin/master -- .gitignore`
> should reset the file to the master version

Thank you, this worked and I've pushed to PR